cfgmgr/teammgr.cpp

```diff
@@ -98,6 +99,43 @@ bool TeamMgr::isLagStateOk(const string &alias)
     return true;
 }
 
+bool TeamMgr::isMACsecSetted(const std::string &port)
```

Comment:
May be change name to isMacsecEnabled

Reply:
This name was in my mind before. But I think `isMACsecEnabled` is more like that MACsec feature has been enabled on the port. But I just want to check whether the MACsec would like to be set on this port. So, I check the config DB instead of state DB in this function.